.eslintrc

```diff
@@ -0,0 +1,26 @@
+
```

Comment:
Is there a way to use this config for prettier? Or switch to using prettier instead of eslint?

Why do we need eslint if we have prettier configured?

Reply:
Usually its good to have both imo.  Prettier is pretty great at guarding against style issues but doesn't really do much for the other things that are necessary: https://prettier.io/docs/en/comparison.html

Reply:
Makes sense! Thanks for the explanation